src/plants/plants.controller.ts

```diff
@@ -53,4 +58,31 @@ export class PlantsController {
       data,
     );
   }
+
+  @Get(':id/water')
+  @ApiOperation({
+    summary: PLANT_WATER_LOG.API_OPERATION.SUMMARY,
+    description: PLANT_WATER_LOG.API_OPERATION.DESCRIPTION,
+  })
+  @ApiParam({
+    type: Number,
+    name: PLANT_WATER_LOG.API_PARAM.NAME,
+    required: true,
+    description: PLANT_WATER_LOG.API_PARAM.DESCRIPTION,
+  })
+  @ApiOkResponse({ type: ResponsePlantWaterLogDto })
+  @ApiBadRequestResponse({
+    description: PLANT_WATER_LOG.ERROR_DESCRIPTION.BAD_REQUEST,
+  })
+  async getPlantWaterLog(
+    @Param() { id }: ReadPlantInformationParam,
```

Comment:
`ReadPlantInformationParam` 은 식물 단계 정보 조회 dto 가 가지고 있는 params 라 이쪽을 참조하기 보다는 
`src/common/dto` 에 공통 dto 로 하나 만들어 두고, extends 해서 쓰는게 좋을 것 같아요

제가 만든 API 에서도 공통 dto 를 가져다 쓰는게 좋을 것 같네요.

Reply:
```ts
// src/common/dto/common-params.dto.ts

export class CommonParams { 
  @IsNumber()
  @Type(() => Number)
  readonly id: number;
}
```
```suggestion
@Param() { id }: CommonParams
```

추후에 다른 params 를 받는 컨트롤러가 있다면 아래처럼 사용할 수 있어서 재사용성을 높일 것 같네요
```ts
export class UnknownDto extends CommonParams {
...
}
```

Reply:
조아요,,